packages/donate-button-beta/src/components/widget/hooks/use-i18n.ts

```diff
@@ -8,14 +8,21 @@ const getBrowserLanguage = () => {
 };
 
 export const getTranslations = (
-	i18n: Record<string, Language>,
+	i18n: Record<string, Partial<Language>>,
 	forceLanguage: string | false
-) => {
+): Language => {
 	const lang = forceLanguage ? forceLanguage : getBrowserLanguage();
 
 	// Language given in config has priority over user browser lang; default to en
 	// since that's always provided by default options
-	return i18n[lang] ?? i18n.en!;
+
+	if (!i18n[lang] || lang === 'en') {
+		return i18n.en as Language;
```

Comment:
why do we need to cast these? seems that these should already return type `Language`.

Reply:
later on an improvement would be to augment this so that specific translations are lazy-loaded, but that's a future improvement.